infra/eligibility_screener/template/ecs.tf

```diff
@@ -0,0 +1,102 @@
+# security group for screener
+resource "aws_security_group" "allow-screener-traffic" {
+  name = "allow_screener_traffic"
+  description = "This rule blocks all traffic unless it is HTTPS for the eligibility screener"
+  vpc_id = "vpc-032e680f92b88bb68" # don't like that this is hardcoded; default vpc
```

Comment:
I would have suggested using `aws_default_vpc`, but I think it gives you a random default VPC which makes it difficult/impossible to put the eligibility screener and API in the same VPC.

You might want to have this in the `constants` module regardless though since both modules need this ID.